src/index.js

```diff
@@ -0,0 +1,171 @@
+import './styles.scss';
+
+import projectFactory from './factory';
+import { initialize, saveData } from './storage';
+import { displayProjects } from './display'; // eslint-disable-line
+
+const initialData = initialize([], 0);
+
+const { projects } = initialData;
+
+let { id } = initialData;
+
+// crating To-do's
+const todoFactory = (title, duedate, desc, note, priority, temp = 'Empty') => {
+  if (temp === 'Empty') {
+    id += 1;
+  } else {
+    id = temp;
+  }
+  return {
+    id, title, duedate, desc, note, priority,
+  };
+};
+
+// delete task from projects
+
+const deleteItem = (task, project) => {
+  const currentProject = projects.find(o => o.name === project.name);
+  currentProject.list = currentProject.list.filter(x => x.id !== task.id);
+
+  saveData(projects, id);
+  displayProjects(projects);
+};
+
+const projectNameList = (list) => {
+  projects.forEach((project) => list.push(project.name));
+  return list;
+};
+
+const validateForm = (title, date, priority) => !(title === '' || date === '' || priority === 'Choose...');
+
+const setAlert = (alert, status) => {
+  alert.style.display = 'block';
```

Comment:
[OPTIONAL] Keep all the styling logic in CSS files . Here for example, you can create class called `.block `in your CSS file then add this class name attribute with JavaScript